src/routes/api/api-reset-subscription-failed-tasks.ts

```diff
@@ -5,7 +5,7 @@ import { TaskType } from "../../sync/sync.types";
 
 interface OutputLogRecord {
 	repoSyncStateId: number,
-	targetTask: "pull" | "branch" | "commit" | "build" | "deployment" | "dependabotAlert" | string
+	targetTask:  string // "pull" | "branch" | "commit" | "build" | "deployment" | "dependabotAlert"
```

Comment:
There is a taskTypes type in `sync.types.ts` that 'should' match this.... but with all the security and code scanning changes it might be out of sync :ohnoes:

Reply:
Yeah maybe this should get updated to keep the stronger typing.  Since it has string in the list it is the only typing used as it overwrites everything else.  I would be happy for us to go to the strong typing but this was function as a comment right now so I just made it into one